app/scripts/lib/router.js

```diff
@@ -219,6 +219,7 @@ function (
         return self.showView(view);
       })
       .fail(function (err) {
+        console.error(err);
```

Comment:
Yea half the time I add it myself to help with debugging. I opened #3025 after looking to see if we have a standard way of using `console.error`, but seems not.